### DISCUSSION TITLE
Can we generate Videos? New type

### DISCUSSION THREAD

**Discussion by Khaos1221** [lllyasviel/Fooocus#2383] [Ideas]
I know many people are like no not right now 
But I mean of frame by frame.  Can we automate a script/ Use pivot animations to make a video and FOOOCUS can develop image over there. I dont mind the time required ,I want to know If  we provide it a frame animation can it recognise and develop upon it.

Edit I have developed full body image and I have developed a Pivot animation/advanced animation. Can FOoocus over lay the images over the animation so that it feel reals

**Comment by mashb1t** [comment]
@Khaos1221 sure, you can input frame by frame to the upscale or vary tab with vary (subtle) or upscale x2

**Reply by Khaos1221** [reply]
thx

**Comment by james-smith-saas** [comment]
Hi ,

" If you have frame-by-frame animation or advanced pivot animations, you can use Fooocus to layer images over the animation to make it feel more realistic. Fooocus mainly focuses on image generation, you can use its upscale and vary features to subtly modify frames or upscale them. It’ll give you more smooth video like feel. 

For more powerful AI video generators like Runway they help transform images into full videos.

LunaBloom mainly provides with text-to-video features.

 And Synthesia works for fine tuned video generation.

If you’re aiming for a more creative approach, Pictory offers great options for turning scripts or short clips into engaging videos.
Deepbrain AI is excellent for creating realistic, AI video content. 
"
